infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/Labels.java

```diff
@@ -99,6 +106,9 @@ public Serializer server(String ref, ServerConfig server) {
       if (server.getPath() != null) {
         labels.put(String.format(SERVER_PATH_LABEL_FMT, ref), server.getPath());
       }
+      if (server.getAttributes() != null) {
```

Comment:
May `server.getAttributes` return null value?

Reply:
Why not? It is just an interface, we don't know which impl is there

Reply:
Ok, I'd propose document that it must not return null value like we do for fields in WorkspaceConfig. As far as I remember there was some declaration that model interfaces must not return null values for collections. If they can we should rework ton of places where we didn't check returned value on null where a collection is returned. For example you can take a look usage of org.eclipse.che.api.core.model.workspace.runtime.Machine#getServers.
Personally, I like such declaration because it makes developers life easier and code cleaner =)